src/conf/statestore.properties

```diff
@@ -16,14 +16,12 @@
 # limitations under the License.
 #
 
-*.domain=${falcon.app.type}
+*.domain=debug
```

Comment:
@sandeepSamudrala  : The statestore.properties in schedule is set to debug mode by default. Thats why I changed it here.